packages/dataviews/src/dataviews-layouts/list/style.scss

```diff
@@ -138,6 +138,10 @@ ul.dataviews-view-list {
 		background-color: $gray-100;
 		border-radius: $grid-unit-05;
 
+		.fields-controls__featured-image-container {
```

Comment:
This is a field specific style, it should live where the field is defined and not in the generic dataviews package.

Reply:
Thank you. Would that be `packages/edit-site/src/components/post-list/style.scss`? A new stylesheet in `/packages/edit-site/src/components/post-fields`? Or somewhere else?

Reply:
> A new stylesheet in /packages/edit-site/src/components/post-fields? 

I think that would be ideal.